docs/redis.md

```diff
@@ -1,96 +1,39 @@
-# `redis` Metrics Receiver
+# Redis
 
-The redis receiver can retrieve stats from your redis server through the [INFO](https://redis.io/commands/info) command. 
-
-
-## Configuration
-
-Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your redis configuration.
-
-To configure a receiver for your redis metrics, specify the following fields:
-
-| Field                 | Default                   | Description |
-| ---                   | ---                       | ---         |
-| `type`                | required                  | Must be `redis`. |
-| `address`             | `localhost:6379`          | The url exposed by redis |
-| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
-| `password`            |                           | The password used to connect to the server. |
-
-Example Configuration:
-
-```yaml
-metrics:
-  receivers:
-    redis_metrics:
-      type: redis
-      address: localhost:6379
-      collection_interval: 30s
-      password: pwd
-  service:
-    pipelines:
-      redis_pipeline:
-        receivers:
-          - redis_metrics
-```
+Follow https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/redis
+for instructions to collect logs and metrics from this application using Ops Agent.
```

Comment:
Follow the [installation guide](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/redis) for instructions to collect logs and metrics from this application using Ops Agent.

Reply:
Sure thing. Changed.